src/sentry/seer/code_review/webhooks/issue_comment.py

```diff
@@ -105,7 +105,8 @@ def handle_issue_comment_event(
         return
 
     if comment_id:
-        _add_eyes_reaction_to_comment(integration, organization, repo, str(comment_id))
+        if options.get("github.webhook.issue-comment"):
```

Comment:
This should be this:
```
if not options.get("github.webhook.issue-comment"):
```

Perhaps it would have been clearer if the option was named: `github.webhook.issue-comment.send_to_overwatch_vs_seer`.